Makefile

```diff
@@ -77,7 +80,7 @@ a11y: test-build pally-conf
 
 # Note: `run` executes `node demo/app`, which fires up express, then deploys
 # a test static site to s3, then exits, freeing the process to execute `nightwatch a11y`.
-test: developer-note verify pally-conf test-server test-browser test-build test-webpack run nightwatch a11y
+test: developer-note verify pally-conf test-server test-browser test-build test-webpack run nightwatch a11y build-webpack
```

Comment:
The CircleCI app preset says to run during test: `make build-production test provision`

I wonder if it'd make better sense to do this instead:

```
build-production: build-webpack
```

and remove the `webpack --bail --config build/deploy/webpack.config.js` build line in the `deploy` target.

So it's in keeping with how we do it elsewhere in Next.

Disclaimer: I've barely worked with `n-ui` and if `n-ui` needs to be different then cool 👍